Palindrome.cs

```diff
@@ -0,0 +1,86 @@
+  using System;
+
+    class Program
+    {
+        /// <summary>
+        /// Determines whether the string is a palindrome.
+        /// </summary>
```

Comment:
The function name give a good description on what it does, so might leave out the comments